server/server.go

```diff
@@ -38,5 +37,21 @@ func main() {
 		return c.JSON(http.StatusOK, res)
 	})
 
+  
+  type Content struct {
+    Username string `json:"username"`
+    ContentID string `json:"id"`
+    ContentBody string `json:"content_body"`
+  }
```

Comment:
This struct should be placed outside of the context of this function similar to the other types.